pkg/client/cache/listers_test.go

```diff
@@ -783,3 +783,90 @@ func TestStoreToServiceLister(t *testing.T) {
 		t.Errorf("Expected service %q, got %q", e, a)
 	}
 }
+
+func TestStoreToLimitRangeLister(t *testing.T) {
```

Comment:
I don't see a whole lot of value to these tests since the logic is done elsewhere.